modyn/models/fmownet/fmownet.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+from torchvision.models import densenet121
+
+
+class FmowNet:
+    # pylint: disable-next=unused-argument
+    def __init__(self, model_configuration: dict[str, Any], device: str, amp: bool) -> None:
+        self.model = FmowNetModel(**model_configuration)
+        self.model.to(device)
+
+
+class FmowNetModel(nn.Module):
```

Comment:
(my comment on some documentation on model architecture + where it is coming from applies to all models, not going to copy it to the other files)

Reply:
I added a link pointing to the paper's implementation of each network